amundsen_application/static/js/components/common/EditableText/index.tsx

```diff
@@ -136,9 +127,21 @@ class EditableText extends React.Component<EditableTextProps, EditableTextState>
                   </div>
                 </Tooltip>
               </Overlay>
-            </div>
           }
-          <div className='editable-text'>{ this.state.value }</div>
+          <div className={"editable-text"}>
+            { this.state.value }
+            <a className={ "edit-link " + (this.state.value ? "" : "no-value") }
```

Comment:
How does this get hidden when `this.state.editable` is `false`? You had mentioned a `hidden` class would be applied.

Reply:
Originally I was using the `hidden` class but encountered a problem with the empty state since `display: none` takes elements out of the DOM flow it became impossible to hover over the div. This problem went away when we decided to have "Add Description" always show, but I had already switched to using `opacity: 0` and I think that works somewhat better since it doesn't affect the flow.

Reply:
I misread the question. I need to cover this case.